src/main/java/Duke/Commands/Delete.java

```diff
@@ -0,0 +1,39 @@
+package Duke.Commands;
+
+import Duke.Commands.Tasks.Task;
+import Duke.TaskList;
+
+/**
+ * This class handles deletion commands
+ */
+public class Delete extends Command {
+    private final int index;
+    private Task removed;
+    private int newSize;
```

Comment:
i like the clear naming, adding 'new' to show that the size var is to be changed